src/richie/apps/courses/migrations/0013_migrate_data_translated_licence_fields.py

```diff
@@ -0,0 +1,49 @@
+# Generated by Django 2.2.8 on 2020-01-02 13:56
+
+from django.conf import settings
+from django.core.exceptions import ObjectDoesNotExist
+from django.db import migrations
+
+
+def forwards_func(apps, schema_editor):
+    Licence = apps.get_model("courses", "Licence")
+    LicenceTranslation = apps.get_model("courses", "LicenceTranslation")
+
+    for object in Licence.objects.all():
+        LicenceTranslation.objects.create(
+            master_id=object.pk,
+            language_code=settings.LANGUAGE_CODE,
+            name=object.name_deprecated,
+        )
+
+
+def backwards_func(apps, schema_editor):
+    Licence = apps.get_model("courses", "Licence")
+    LicenceTranslation = apps.get_model("courses", "LicenceTranslation")
+
+    for object in Licence.objects.all():
+        translation = _get_translation(object, LicenceTranslation)
+        object.name_deprecated = translation.name
+        object.save()  # Note this only calls Model.save()
+
+
+def _get_translation(object, LicenceTranslation):
+    translations = LicenceTranslation.objects.filter(master_id=object.pk)
+    try:
+        # Try default translation
+        return translations.get(language_code=settings.LANGUAGE_CODE)
+    except ObjectDoesNotExist:
+        try:
+            # Try default language
+            return translations.get(language_code=settings.PARLER_DEFAULT_LANGUAGE_CODE)
+        except ObjectDoesNotExist:
+            # Maybe the object was translated only in a specific language?
+            # Hope there is a single translation
+            return translations.get()
```

Comment:
Why not get the first one? 
```suggestion
            return translations.first()
```